addons/mail/static/src/core/common/message_reactions.js

```diff
@@ -60,7 +60,7 @@ export class MessageReactions extends Component {
     }
 
     hasSelfReacted(reaction) {
-        return this.store.self.in(reaction.personas);
+        return this.store.self?.in(reaction.personas);
```

Comment:
When can `store.self` be nullish? I thought when not logged with internal user, you are authenticated as a guest in Discuss code?

Reply:
Portal users without login (with token) are authenticated by their token. In this case, `store.self` is `null`.
Maybe after finalizing the ACL for this PR, it will change?